### PR TITLE
[dom daint] Upgrade PLUMED

### PR DESCRIPTION
**Pull request by lucamar**
Provide the latest stable release of PLUMED (2.6.1) in production.

### CODE REVIEW

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.1-CrayGNU-20.08.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'CrayGNU', 'version': '20.08'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = ['https://github.com/%(namelower)s/plumed2/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(namelower)s-compiler.patch']
```
Comment (victorusu) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.6.1-CrayGNU-20.08.eb:21]: Do we really need this patch?

**Comment by lucamar** [comment]
Thanks for checking @victorusu, I have removed the patch from the `CrayGNU` easyconfig file, since the build is configured properly with the `preconfigopts` and the executable is linked correctly with the `prebuildopts`.

**Comment by lucamar** [comment]
I have also moved to the `deprecated` folder the outdated easyconfig files, so it will be easier for users to find the current recipes.

**Review by victorusu — APPROVED** [review]
`lgtm`

**Pull request merged**